lib/src/modules/preview/PreviewImage.js

```diff
@@ -58,9 +59,7 @@ function PreviewImage({
           {imageHasFailed && fallback}
         </div>
 
-        <div className="absolute top-0 m-2">
-          <div className="preview-children">{children}</div>
-        </div>
+        {children}
```

Comment:
controls are not named and handled in file cards. This is just good practice to render `children` still 👍